javascript/ql/src/Declarations/UnusedVariable.ql

```diff
@@ -54,19 +54,23 @@ predicate isPropertyFilter(UnusedLocal v) {
  * references it.
  */
 predicate isReactImportForJSX(UnusedLocal v) {
```

Comment:
Actually, why is this specific to imports? Maybe we should just lift that restriction entirely.

As far as I understand, the JSX compiler doesn't care if the name comes from an import, it will just use that variable name.

Reply:
@xiemaisi do you have an opion on this? I have pushed a commit that does just that, but I need to evaluate it.

Reply:
I like the simplification a lot, but as you say we'll need at least a small evaluation to ensure the `getParent+()` behaves itself.